.github/workflows/deploy-hubs.yaml

```diff
@@ -180,32 +180,33 @@ jobs:
     #
     # If you are adding a new cluster, please remember to list it here!
     outputs:
-      failure_2i2c: "${{ env.failure_2i2c }}"
+      failure_2i2c-aws-us: "${{ env.failure_2i2c-aws-us }}"
       failure_2i2c-uk: "${{ env.failure_2i2c-uk }}"
+      failure_2i2c: "${{ env.failure_2i2c }}"
```

Comment:
I sorted these alphabetically, makes it less likeley that we get merge conflicts compared to appending - but mostly because its easier to read.